crates/driver/src/api/mod.rs

```diff
@@ -1,42 +1,56 @@
-pub mod execute;
-pub mod solve;
-
 use {
-    crate::driver::Driver,
+    crate::solver::Solver,
     futures::Future,
-    shared::api::finalize_router,
+    nonempty::NonEmpty,
     std::{net::SocketAddr, sync::Arc},
-    tokio::{task, task::JoinHandle},
-    warp::{Filter, Rejection, Reply},
 };
 
-pub fn serve_api(
-    address: SocketAddr,
-    shutdown_receiver: impl Future<Output = ()> + Send + 'static,
-    drivers: Vec<(Arc<Driver>, String)>,
-) -> JoinHandle<()> {
-    let filter = handle_all_routes(drivers).boxed();
-    tracing::info!(%address, "serving driver");
-    let (_, server) = warp::serve(filter).bind_with_graceful_shutdown(address, shutdown_receiver);
-    task::spawn(server)
+pub mod execute;
+pub mod info;
+pub mod solve;
+
+const REQUEST_BODY_LIMIT: usize = 32 * 1024;
```

Comment:
I think at the moment we don't have a limit because the solver binary fetches the current auction instead of having a route like /solve .

Considering this will only be called by a trusted party (autopilot) we might not need a limit.

Reply:
For `/solve` I think it would have to be in the MBs. For other requests, the 32KB should be fine.